README.md

```diff
@@ -13,4 +13,12 @@ You can install `circomlibjs` with the following command:
 
 ```text
 npm install -g circomlibjs
-```
\ No newline at end of file
+```
+
+
+#Forked off of iden3/circomlibjs
+This fork has been changed to support Starkware version of Poseidon3 contract for ethereum L1. Current version has been deployed on goerli at 0x84d43a8cbEbF4F43863f399c34c06fC109c957a4. Byte code and abi included in the deployed folder. The updated implementation is reflected in Javascript(Poseidon_refernce.js) and EVM/Solidity OpCodes (podeidon_gencontract.js)
```

Comment:
typos: 
- `src/poseidon_reference.js` instead of `Poseidon_refernce.js`
- `src/poseidon_gencontract.js` instead of `podeidon_gencontract.js`

Reply:
you could add a link to Goerli Etherscan while mentionning the address

Reply:
could we verify the contract?

Reply:
so they are EVM codes not solidity, I will try again to verify it to at least have it accept the ABI